File: Euclid's Algorithm/Go/Eulic's-Algorithm.go
package main

import "fmt"

func gcd(a int, b int) int {
    if b == 0 {
        return a
    }
    return gcd(b, a % b)
}

func main() {
    a := 48
    b := 18
    fmt.Printf("GCD of %d and %d is %d\n", a, b, gcd(a, b))
}

File: Euclid's Algorithm/C/Eulic's-Algorithm.c
#include <stdio.h>

int gcd(int a, int b) {
    if (b == 0)
        return a;
    else
        return gcd(b, a % b);
}

int main() {
    int a = 60, b = 48;
    printf("GCD of %d and %d is %d\n", a, b, gcd(a, b));
    return 0;
}
